src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -691,6 +701,20 @@ theorem shiftLeft_shiftLeft {w : Nat} (x : BitVec w) (n m : Nat) :
     (x <<< n) <<< m = x <<< (n + m) := by
   rw [shiftLeft_add]
 
+/-! ### shiftLeft reductions from BitVec to Nat -/
+
+theorem shiftLeft_eq' {x : BitVec w} {y : BitVec w₂} :
+  x <<< y = x <<< y.toNat := by rfl
```

Comment:
Can this just be a simp lemma, and then we never prove anything about shifting by a BitVec?